code/index.html

```diff
@@ -3,20 +3,117 @@
   <head>
     <meta charset="utf-8" />
     <title>Project 2: The News Site</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <!--external CSS connection-->
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <!-- meta tag for responsiveness -->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
   </head>
   <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
+<!--menu-->
+<header class="mobilemenu">
+	<div class="dropdownlogo" style="float:left;">
```

Comment:
Look over the indentation on this. So that the `header`-tag follows under the `body`-tag and  so the `div`-tag follows the header.